src/Configuration/ConfigurationValueProxy.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Class ConfigurationProxy a simple wrapper that allows passing a pointer to the eventual
+ * compiled and validated configuration.
+ * @package Bolt\Configuration
+ * @author Ross Riley <riley.ross@gmail.com>
+ */
```

Comment:
G-pick … could you push that down on to of the class declaration and drop the `@package`